aws/src/test/java/org/frankframework/filesystem/AmazonS3FileSystemTestHelper.java

```diff
@@ -131,7 +131,7 @@ public boolean _fileExistsInBucket(String objectName, String bucket) {
 		try {
 			HeadObjectResponse headObjectResponse = s3Client.headObject(HeadObjectRequest.builder()
 					.key(objectName)
-					.bucket(bucketName)
+					.bucket(bucket)
```

Comment:
I would suggest changing parameter name to `bucketName` to match `objectName`, and then rename the field `bucketName` to `defaultBucketName`. That would make such mistakes less likely.